runtime/src/main/java/io/quarkiverse/logging/cloudwatch/LoggingCloudWatchConfig.java

```diff
@@ -109,19 +109,29 @@ public class LoggingCloudWatchConfig {
     @ConfigItem
     public Optional<Duration> apiCallTimeout;
 
+    /**
+     * Default credentials provider enabled added as a {@code credentials.provider.type.default}
+     */
+    @ConfigItem(name = "credentials.provider.type.default", defaultValue = "false")
```

Comment:
Let's match the usual naming convention for config properties:
```suggestion
    @ConfigItem(name = "quarkus.log.cloudwatch.default-credentials-provider.enabled", defaultValue = "false")
```